packages/@react-spectrum/dialog/test/DialogTrigger.test.js

```diff
@@ -965,7 +965,7 @@ describe('DialogTrigger', function () {
       expect(outerDialog).toBeVisible();
     }); // wait for animation
     let innerButton = getByTestId('innerButton');
-    userEvent.tab();
+    act(() => innerButton.focus());
```

Comment:
Appears to be a bug in user-event, where if tabbing from an element with `tabindex="-1"` (as in our dialog elements), an incorrect next element is focused.

Reply:
Can we add an inline comment here?